website/cue/reference/components/sinks.cue

```diff
@@ -287,7 +287,7 @@ components: sinks: [Name=string]: {
 										enum: {
 											bytes:               "Byte frames are concatenated."
 											character_delimited: "Byte frames are delimited by a chosen character."
-											length_delimited:    "Byte frames are encoded by adding a length header."
+											length_delimited:    "Byte frames are prefixed by an unsized big-endian 32-bit integer indicating the length."
```

Comment:
I think you meant _unsigned_ here, not _unsized_? 🙈 
Please don't hate me 😄

Reply:
😂 you are right.

Reply:
https://github.com/vectordotdev/vector/pull/13491

Reply:
I would have sent a PR myself but I'm away from the computer today hehe.